functions/groups.js

```diff
@@ -283,6 +283,48 @@ const StartTime = {
   }
 }
 
+const RoundStartTime = {
+  name: 'RoundStartTime',
+  docs: 'The start time of a round. If a round exists multiple times, return the min of the start times',
```

Comment:
nit: "The start time of the first group of a round"? I know what you mean by "exists multiple times" but it's a little confusing (a round can't actually happen multiple times, it just appears multiple times on the schedule).

Ditto for RoundEndTime

Reply:
Hum actually it uses the "lowest" start time of all the occurrences of the round in the schedule; it's used before any group is created as I intended to use that specifically to automatically detect the start time of the first groups (and the end time of the last groups).

Does it still look good to you with that behavior?

I'll absolutely update the docs to clarify the intention!

Reply:
Yeah, the behavior sounds right to me. I just think my suggested docs is clearer and simpler :)

Reply:
Rebased and updated, sorry for the delay on making this very small change ^^